addons/binding/org.openhab.binding.edimax/src/main/java/org/openhab/binding/edimax/internal/EdimaxHandlerFactory.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.edimax.internal;
+
+import static org.openhab.binding.edimax.EdimaxBindingConstants.*;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.edimax.handler.Edimax1101Handler;
+import org.openhab.binding.edimax.handler.Edimax2101Handler;
+
+/**
+ * The {@link EdimaxHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Falk Harnisch - Initial contribution
+ */
+public class EdimaxHandlerFactory extends BaseThingHandlerFactory {
+
+    private final static Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = new HashSet<>(
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed recently) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059